main.js

```diff
@@ -3,46 +3,58 @@ const author = document.querySelector('.author');
 const form = document.querySelector('form');
 const displayBooks = document.querySelector('.container');
 const message = document.querySelector('span');
+
 // Is localStorage Empty
 if (localStorage.getItem('Book List') === null) {
   localStorage.setItem('Book List', JSON.stringify([]));
 }
+
 // Data Storage into localStorage
 const dataStore = JSON.parse(localStorage.getItem('Book List'));
 // localStorage data update
 function updateData() {
   localStorage.setItem('Book List', JSON.stringify(dataStore));
 }
-// Create data row
+
 function createBooks(items) {
   let books = '';
   for (let i = 0; i < items.length; i += 1) {
     books += `
-      <p>${items[i].title}</p>
-      <p>${items[i].author}</p>
-      <button onclick='removeitem(${i})'>Remove</button>
-      <hr />
-    `;
+      <div class="displayRow">
+        <ul>
+          <li>${items[i].title} by ${items[i].author}</li>
+        </ul>
+        <button onclick='removeitem(${i})'>Remove</button>
+      </div>
+      <hr/>
+      `;
```

Comment:
- [ ] [Optional] You can also add this function as a method inside `NewBook` class you created.